content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -137,20 +152,62 @@ use, see `man 7 user_namespaces`.
 
 ## Set up a node to support user namespaces
 
-It is recommended that the host's files and host's processes use UIDs/GIDs in
-the range of 0-65535.
+By default, the kubelet assumes host's files and host's processes use UIDs/GIDs
+in the range of 0-65535. This is the case in most Linux distributions with the
+default installation. The kubelet will assign UIDs/GIDs higher than that to
+pods, so there is no overlap between UID/GIDs used by host's files and processes
+and the ones used by pods.
 
-The kubelet will assign UIDs/GIDs higher than that to pods. Therefore, to
-guarantee as much isolation as possible, the UIDs/GIDs used by the host's files
-and host's processes should be in the range 0-65535.
-
-Note that this recommendation is important to mitigate the impact of CVEs like
+Avoiding the overlap is important to mitigate the impact of CVEs like
```

Comment:
Try this:
```suggestion
Avoiding the overlap is important to mitigate the impact of vulnerabilities such as
```
CVE is actually a database, and in docs it helps to be more precise than you would in speech or in a blog article